include/jemalloc/internal/jemalloc_internal_externs.h

```diff
@@ -34,6 +34,7 @@ extern malloc_init_t malloc_init_state;
 extern const char *zero_realloc_mode_names[];
 extern atomic_zu_t zero_realloc_count;
 extern bool opt_cache_oblivious;
+extern unsigned opt_debug_dblfree_cbin_scan_limit;
```

Comment:
Maybe `opt_debug_double_free_max_scan`

Reply:
addressed in [5a2502a](https://github.com/jemalloc/jemalloc/pull/2310/commits/5a2502a2dd9fb3e401373e087983f334ed35dbc4)